src/deepsparse/pipeline.py

```diff
@@ -467,7 +472,7 @@ def to_config(self) -> "PipelineConfig":
         )
 
     def join_engine_outputs(
-        self, batch_outputs: List[List[numpy.ndarray]], orig_batch_size: int
+        self, batch_outputs: List[List[numpy.ndarray]], orig_batch_size: int, **kwargs
```

Comment:
should the kwargs be forwarded to the `return join_engine_outputs(batch_outputs, orig_batch_size)`?

Reply:
not relevant for anything beyond text generation atm